src/styles/watch.scss

```diff
@@ -0,0 +1,61 @@
+.watch {
+  position: absolute;
+  top: 50%;
+  left: 50%;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.